dotcom-rendering/src/web/components/TextBlockComponent.tsx

```diff
@@ -35,33 +35,43 @@ const isOpenQuote = (t: string): boolean => {
 	].includes(t);
 };
 
-const isLongEnough = (html: string): boolean => {
-	// Only show a dropcap if the block of text is 200 characters or
-	// longer. But we first need to strip any markup from our html string so
-	// that we accurately measure the length that the reader will see. Eg. remove
-	// link tag html.
+const stripHtmlFromString = (html: string): string => {
```

Comment:
We already use `jsdom` in many other places. Should we not simply parse this paragraph and use the node's `textContent`?

Reply:
It does feel like it, but might move the logic further away from the effect it has. Been trying to work out a good shape for it. Happy to pair if you fancy!

Reply:
Do we want to be cautious about using JSDOM in components, as they are sometimes also used client-side? We probably want to avoid shipping JSDOM to browsers (which have their own APIs for parsing HTML into DOM trees).